contracts/HashRegistrarSimplified.sol

```diff
@@ -305,16 +307,24 @@ contract Registrar {
 
     /**
      * @dev Start multiple auctions for better anonymity
+     *
+     * Anyone can start an auction by sending an array of hashes that they want to bid for.
+     * Arrays are sent so that someone can open up an auction for X dummy hashes when they
+     * are only really interested in bidding for one. This will increase the cost for an
+     * attacker to simply bid blindly on all new auctions. Dummy auctions that are
+     * open but not bid on are closed after a week.
+     *
```

Comment:
This information should be attached with this bulk version of `startAuctions` rather than the single version above.